app/queries/colleague_trainees_recent_reports_query.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ColleagueTraineesRecentReportsQuery < Patterns::Query
+  queries Report
+
+  private
+
+  def query
+    relation
+      .joins(:user)
+      .where(users: { id: colleague_trainee_ids })
+      .where(wip: false)
+      .default_order
+      .includes(user: { avatar_attachment: :blob })
```

Comment:
```suggestion
      .not_wip
      .default_order
      .with_avatar
```
`default_order`と同じ様に、モデルとモジュールのスコープを再利用してDRYにしてもいいのではと思いましたがどうでしょうか👀?

Reply:
確かにそうでした。
Wikiのコードをそのまま書いていて確認不足でした💦
修正します🙏